### ISSUE TITLE
Dark Chat Theme Broken On Latest Version

### ISSUE THREAD

**Issue by garv-shah** [flyerhq/flutter_chat_ui#197]
# General

## What bug do you experience? 🐞
If you replace the default chat theme with the dark chat theme, you get a slightly broken message bar, where the sizing is a bit wrong. This is most likely due to the few changes in https://github.com/flyerhq/flutter_chat_ui/commit/f4ff7f89b09ae42622e64777ffe856f46e6db9cc not being applied to the Dark Chat Theme as well.

## How can it be reproduced? 🤔
Replace the chat widget's theme property with DarkChatTheme()

## What behavior is expected? 💡
The same padding as the default chat theme with different colours

<br />

# Extras

## Screenshots or videos 📸
![IMG_0460](https://user-images.githubusercontent.com/64292655/150301108-1a1f4be5-435e-4ddf-a851-f0f70db68730.jpg)

## Code snippets 📝
If applicable, add code samples to help explain your problem.
I'm pretty sure it's a pretty easy fix, i think the inputPadding and inputMargin just got mixed up, so the UI looks normal if u just swap them by providing the values below

```dart
const DarkChatTheme(inputPadding: EdgeInsets.fromLTRB(24, 20, 24, 20), inputMargin: EdgeInsets.zero)
```

<br />

# Environment info

Please specify the flutter, flutter-chat-ui versions.

flutter: 2.8.1
flutter-chat-ui: 1.5.4

`flutter doctor -v` output 👇

```sh
[✓] Flutter (Channel stable, 2.8.1, on macOS 12.1 21C52 darwin-arm, locale en-AU)
    • Flutter version 2.8.1 at /Users/garvshah/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (5 weeks ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/garvshah/Library/Android/sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: /Users/garvshah/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7935034/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/garvshah/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7935034/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/garvshah/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7784292/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.3)
    • IntelliJ at /Users/garvshah/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/213.5744.223/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] IntelliJ IDEA Ultimate Edition (version 2021.2.2)
    • IntelliJ at /Users/garvshah/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/212.5284.40/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] Connected device (4 available)
    • SM G977B (mobile)      • R3CM506K7YF               • android-arm64  • Android 11 (API 30)
    • Garv’s iPhone (mobile) • 00008030-000A35492690802E • ios            • iOS 15.2 19C56
    • macOS (desktop)        • macos                     • darwin-arm64   • macOS 12.1 21C52 darwin-arm
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 97.0.4692.71
    ! Error: Garv’s iPhone is busy: Making Garv’s iPhone ready for development. Xcode will continue when Garv’s iPhone is finished. (code -10)

• No issues found!
```

<br />

# Platform

Device (e.g. Android emulator, iOS simulator, Samsung Galaxy S21): all

OS version (e.g. iOS 14.5, Android 11): all

<br />

# Additional context

<!-- Thank you for making us better 🤝--!>

**Comment by demchenkoalex** [comment]
Thanks! Will fix. Or you can also open a PR, I guess I forgot to apply paddings :D

**Comment by garv-shah** [comment]
sure!

**Comment by garv-shah** [comment]
https://github.com/flyerhq/flutter_chat_ui/pull/198
done :)

**Comment by demchenkoalex** [comment]
Released in [v1.5.5](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.5.5).
